src/healthRecord/dto/healthRecord.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsArray, IsDate, IsObject, IsString } from 'class-validator';
+
+export class HealthDataDto {
+  @ApiProperty()
+  @IsDate()
+  dateTime: Date;
+
+  @ApiProperty()
+  @IsArray()
+  values: Array<string>;
+}
+
+export class HealthRecordDto {
```

Comment:
Can some other names be used for this class? I need to use the name 'HealthRecordDto'